src/utils/fileUtils.js

```
@@ -79,16 +79,27 @@ class FileUtils {
             const jsonFilePath = path.join(outputDir, `${baseFilename}.json`);
             fs.writeFileSync(jsonFilePath, JSON.stringify(data, null, 2));
             console.log(ColorUtils.green(`JSON report saved to: ${ColorUtils.yellow(jsonFilePath)}`));
+            this.checkFileSize(jsonFilePath);
         }        // Save HTML output  
         if (format === 'html' || format === 'all') {
             const HtmlFormatter = require('../formatters/htmlFormatter');
             const htmlFilePath = path.join(outputDir, `${baseFilename}.html`);
             fs.writeFileSync(htmlFilePath, HtmlFormatter.generate(data));
             console.log(ColorUtils.green(`HTML report saved to: ${ColorUtils.yellow(htmlFilePath)}`));
+            this.checkFileSize(htmlFilePath);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)